src/logger.ts

```diff
@@ -0,0 +1,86 @@
+import path from 'path';
+import pino from 'pino';
+import pinoHttp from 'pino-http';
+import { loggerConfig } from '../logger.config';
+
+const {
+  moduleFilter, prefixFilter, logLevel, httpLogLevel,
+} = loggerConfig;
+
+const formatPrefix = (prefix: string): string => `[${prefix.toUpperCase()}] `;
+const PROJECT_ROOT = path.resolve(__dirname, '..');
+
+const filterPrefix = (config: {
+  options: pino.ChildLoggerOptions;
+  bindings: pino.Bindings;
+}): boolean => {
+  if (prefixFilter.length === 0) {
+    return true;
+  }
+  const { options } = config;
+  const { msgPrefix } = options;
+  const prefix = formatPrefix(msgPrefix || '');
+  if (prefix && prefixFilter.map((p) => formatPrefix(p)).includes(prefix)) {
+    return true;
+  }
+  return false;
+};
+
+const filterModule = (config: {
+  options: pino.ChildLoggerOptions;
+  bindings: pino.Bindings;
+}): boolean => {
+  if (moduleFilter.length === 0) {
+    return true;
+  }
+  const { bindings } = config;
+  const { module: mod } = bindings;
+  if (mod && moduleFilter.includes(mod)) {
+    return true;
+  }
+  return false;
+};
+
+const FILTERS = [filterModule, filterPrefix];
+
+const doesPassFilters = (config: {
+  options: pino.ChildLoggerOptions;
+  bindings: pino.Bindings;
+}): boolean => FILTERS.every((f) => f(config));
+
+export const expressLogger = pinoHttp({
+  level: httpLogLevel,
+  msgPrefix: '[EXPRESS]: ',
```

Comment:
Why does the `msgPrefix` here have a different format than all other ones, ie square brackets, space, etc?

Reply:
i did not call `formatPrefix` there but instead hardcoded the value.